specification/apicenter/ApiCenter.DataApi/examples/2024-02-01-preview/ApiDefinitions_GetExportSpecificationOperationStatus.json

```diff
@@ -0,0 +1,23 @@
+{
+  "title": "ApiDefinitions_GetExportSpecificationOperationStatus",
+  "operationId": "ApiDefinitions_GetExportSpecificationOperationStatus",
+  "parameters": {
+    "api-version": "2024-02-01-preview",
+    "workspaceName": "default",
+    "apiName": "echo-api",
+    "versionName": "2023-01-01",
+    "definitionName": "default",
+    "operationId": "00000000-0000-0000-0000-000000000001"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "status": "Succeeded",
+        "result": {
+          "value": "{ ... }",
+          "format": "inline"
```

Comment:
@azaslonov Now that Data API export spec operation only returns link, should we change this example to say "link" rather than "inline"?